applications/views.py

```diff
@@ -98,9 +99,14 @@ def applications_csv(request, city):
     response = HttpResponse(content_type='text/csv')
     response['Content-Disposition'] = u'attachment; filename="{}.csv"'.format(city)
     writer = csv.writer(response)
-    writer.writerow(["Application Number", "email", "Application State", "RSVP Status", "Average Score"])
+    csv_header = ["Application Number","Application State", "RSVP Status", "Average Score"]
+    # replace commas with semicolons so it won't confuse csv parsers
+    csv_header.extend([striptags(question).replace(',', ';') for question in Question.objects.values_list('title', flat=True)])
```

Comment:
This gets you all the questions... it needs to be restricted to just that application form. (try having two events in the system and see what I mean :smile:)

I am also not sure about the commas needing to be replaced... I think `csv` takes care of this and surrounds it with quotes.